src/Doctrine/ORM/RefundPaymentRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\RefundPlugin\Repository\RefundPaymentRepositoryInterface;
+
+class RefundPaymentRepository extends EntityRepository implements RefundPaymentRepositoryInterface
```

Comment:
Should we (similar to factories) provide trait that should be implemented in the repository? But I suppose it's not a case for this change but for all the repositories and plugin 💃

Reply:
I think so, but this is the plugin, that defines RefundPayment, so creating a repository without traits makes sense here.